ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/dummy/ExceptionsIT.scala

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.suites
+
+import com.daml.ledger.api.testtool.infrastructure.LedgerTestSuite
+
+// Empty dummy suite for LF versions that do not support exceptions.
```

Comment:
If someone has a nicer suggestion here, I’m all ears. This isn’t super nice but it works and is somewhat extensible to more tests that are version specific.

Reply:
It might be worth adding a "noop" test case with a test title like "empty test cases because DAML LF under test doesn't support exceptions".

Reply:
I’m not quite following. Users should never be able to run this. We just need to get this to compile but we don’t add it to the list of available test cases for an LF version that does not support it.